man/fi_endpoint.3.md

```diff
@@ -472,6 +472,11 @@ The following option levels and option names and parameters are defined.
   exchange between peer endpoints using the fi_connect, fi_accept,
   and fi_reject operations.  This option is read only.
 
+  The available space in CM messages may differ between endpoints created from
+  different domains. The size returned reflects the properties of the domain
+  from which the endpoint was created, except in the case of passive endpoints,
+  in which the size reflects the maximum size across all domains.
```

Comment:
I would drop the references to the domain from this definition.  For active endpoints, the size is the maximum size of CM data for that endpoint in connect, accept, and reject calls.  I would not make any implied association with that size and the size used by other endpoints in the domain.  It's conceivable (though unlikely) that the sizes could differ based on the EP properties.  For passive endpoints, the size is the maximum size of data that may be present as part of a connection request.  Again, drop any reference to the domain and instead explicitly define what the size relates to.